test/integration/model/SharedStimulusPackageImporterTest.php

```diff
@@ -37,14 +38,20 @@
 class SharedStimulusPackageImporterTest extends TestCase
 {
     /**
-     * @var MockObject
+     * @var MediaService|MockObject
      */
-    private $service = null;
+    private $mediaServiceMock;
+    /**
+     * @var StoreService|MockObject
+     */
+    private $storeServiceMock;
```

Comment:
Blank lines required according to our code guideline

```suggestion
    private $mediaServiceMock;
    
    /* @var StoreService|MockObject */
    
    private $storeServiceMock;
```